src/main/java/com/blackducksoftware/integration/hub/alert/accumulator/RealTimeListener.java

```diff
@@ -43,21 +46,24 @@
     private final static Logger logger = LoggerFactory.getLogger(RealTimeListener.class);
 
     private final ChannelTemplateManager channelTemplateManager;
-    private final DigestNotificationProcessor notificationProcessor;
+    private final ProjectDataFactory projectDataFactory;
+    private final NotificationEventManager eventManager;
 
     @Autowired
-    public RealTimeListener(final Gson gson, final ChannelTemplateManager channelTemplateManager, final DigestNotificationProcessor notificationProcessor) {
+    public RealTimeListener(final Gson gson, final ChannelTemplateManager channelTemplateManager, final ProjectDataFactory projectDataFactory, final NotificationEventManager eventManager) {
         super(gson, RealTimeEvent.class);
         this.channelTemplateManager = channelTemplateManager;
-        this.notificationProcessor = notificationProcessor;
+        this.projectDataFactory = projectDataFactory;
+        this.eventManager = eventManager;
     }
 
     @JmsListener(destination = RealTimeEvent.TOPIC_NAME)
     @Override
     public void receiveMessage(final String message) {
         try {
             final RealTimeEvent event = getEvent(message);
-            final List<AbstractChannelEvent> events = notificationProcessor.processNotifications(DigestTypeEnum.REAL_TIME, event.getNotificationList());
```

Comment:
Is this used at all (now that it's deleted)?

Reply:
Or is it used in daily, and we just don't want to remove notifications from the real time digest?

Reply:
Its used in daily, we dont want to remove deletions from real time. We can have a longer conversation about the processing if you'd like